app/templates/Event/_common/event_attendees.html.twig

```diff
@@ -0,0 +1,17 @@
+{% if (event.getAttendeeCount  > 0) %}
+<h3>Who's attending?</h3>
+    <div class="row">
+        {% for attendee in attendees %}
+            <div class="col col-sm-6 col-xs-4">
```

Comment:
I would suggest adding a `text-center' class to this `<div>`, it will look a lot nicer!